lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/experianKbvCri/ContractTest.java

```diff
@@ -0,0 +1,1041 @@
+                     }
+                   ]
+                 }
+               ]
+               }
+             }
+            """;
+
+    private static final String FAILED_VC_BODY =
+            """
+            {
+              "iss": "dummyExperianKbvComponentId",
+              "sub": "test-subject",
+              "nbf": 4070908800,
+              "vc": {
+                "type": [
+                  "VerifiableCredential",
+                  "IdentityCheckCredential"
+                ],
+                "credentialSubject": {
+                  "name": [
+                    {
+                      "nameParts": [
+                        {
+                          "type": "GivenName",
+                          "value": "Mary"
+                        },
+                        {
+                          "type": "FamilyName",
+                          "value": "Watson"
+                        }
+                      ]
+                    }
+                  ],
+                  "birthDate": [
+                    {
+                      "value": "1932-02-25"
+                    }
+                  ],
+                  "address": [
+                        {
+                          "uprn": "10022812929",
+                          "organisationName": "FINCH GROUP",
+                          "subBuildingName": "UNIT 2B",
+                          "buildingNumber": "16",
+                          "buildingName": "COY POND BUSINESS PARK",
+                          "dependentStreetName": "KINGS PARK",
+                          "streetName": "BIG STREET",
+                          "doubleDependentAddressLocality": "SOME DISTRICT",
+                          "dependentAddressLocality": "LONG EATON",
+                          "addressLocality": "GREAT MISSENDEN",
+                          "postalCode": "HP16 0AL",
+                          "addressCountry": "GB"
+                        }
+                      ]
+                },
+                "evidence": [
+                    {
+                        "type": "IdentityCheck",
+                        "txn": "dummyTxn",
+                        "verificationScore": 0,
+                        "checkDetails": [
+                            {
+                                "checkMethod": "kbv",
+                                "kbvQuality": 3,
+                                "kbvResponseMode": "free_text"
+                            }
+                        ],
+                        "failedCheckDetails": [
+                            {
+                                "kbvResponseMode": "free_text",
+                                "checkMethod": "kbv"
+                            },
+                            {
+                                "kbvResponseMode": "multiple_choice",
+                                "checkMethod": "kbv"
+                            }],
+                            "ci": ["V03"]
```

Comment:
Whilst it is OK to use CI codes with no context, we should default to using fake codes unless the provider can't produce VCs with fake codes.